packages/store/ts/config/v2/generics.ts

```diff
@@ -0,0 +1,7 @@
+export type get<input, key> = key extends keyof input ? input[key] : undefined;
```

Comment:
is `undefined` here desirable?

Reply:
yeah, it behaves the same way as accessing a non-existent property in js and allows us to handle that case in typeland too (as opposed to sth like `never`)